pulpcore/cli/common/generic.py

```diff
@@ -842,7 +828,7 @@ def _type_callback(ctx: click.Context, param: click.Parameter, value: Optional[s
     help=_(
         "JSON dictionary of labels to set on {entity} (or " "@file containing a JSON dictionary)"
     ),
-    callback=labels_callback,
+    callback=load_json_callback,
```

Comment:
The existing `labels_callback` provides valuable validation. Please keep it.

Reply:
I thought you wanted to remove it by saying this: https://github.com/pulp/pulp-cli/pull/608#discussion_r1049673939. My bad. I did this now:
```

def load_labels_callback(ctx: click.Context, param: click.Parameter, value: str) -> Any:
    value = load_json_callback(ctx, param, value)

```

Reply:
So labels are optionally read from file and always validated? Sounds right to me.